test/e2e/mock-e2e.js

```diff
@@ -190,6 +190,54 @@ async function setupMocking(server, testSpecificMock, { chainId }) {
       };
     });
 
+  // Both are added to support swaps e2e tests
+  await server
+    .forGet(`${GAS_API_BASE_URL}/networks/1/gasPrices`)
+    .thenCallback(() => {
+      return {
+        statusCode: 200,
+        json: {
+          SafeGasPrice: '1',
+          ProposeGasPrice: '2',
+          FastGasPrice: '3',
+        },
+      };
+    });
+  await server
```

Comment:
Rather than just duplicating these, can we put the existing ones in a method so we mock both chain IDs without the duplication?

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/23717/commits/a5df6919ec254d2f24ca3d521485958f8c9fa5bf